apps/total-typescript/src/components/app/navigation.tsx

```diff
@@ -70,38 +68,175 @@ const DesktopNav = () => {
           <FireIcon className="h-5 w-5 text-orange-400" aria-hidden="true" />
         }
       />
+      {isSellingLive && (
+        <hr
+          className="mx-3 h-1/4 w-px border-transparent bg-gray-700"
+          aria-hidden="true"
+        />
+      )}
+      {canViewContent ? (
+        <AccountDropdown />
+      ) : (
+        isSellingLive && <NavLink path="/login" label="Log in" />
+      )}
     </ul>
   )
 }
 
+const MobileNav = () => {
+  const ability = useAbilities()
+  const canViewTeam = ability.can('view', 'Team')
+  const canViewContent = ability.can('view', 'Content')
+  const canViewInvoice = ability.can('view', 'Invoice')
+
+  return (
+    <div className="block md:hidden">
+      <ul className="flex h-full items-center justify-center">
+        {isSellingLive && !canViewContent && (
+          <NavLink path="/login" label="Log in" />
+        )}
+        <li className="h-full">
+          <NavigationMenu.Root delayDuration={0} className="flex h-full">
+            <NavigationMenu.List className="flex h-full items-center justify-center">
+              <NavigationMenu.Item className="h-full">
+                <NavigationMenu.Trigger className="-mr-3 flex h-full items-center justify-center px-5 hover:bg-gray-800 sm:-mr-5">
+                  <MenuIcon className="h-4 w-4" aria-hidden />
+                  <span className="sr-only">Menu</span>
+                </NavigationMenu.Trigger>
+                <NavigationMenu.Content className="absolute left-0 top-full w-full bg-gray-800 shadow-xl">
+                  <ul>
+                    <MobileNavLink
+                      path="/tutorials"
+                      label="Free Tutorials"
+                      icon={
+                        <PlayIcon
+                          className="h-5 w-5 text-cyan-300"
+                          aria-hidden="true"
+                        />
+                      }
+                    />
+                    <MobileNavLink
+                      path="/tips"
+                      label="Tips"
+                      icon={
+                        <FireIcon
+                          className="h-5 w-5 text-orange-400"
+                          aria-hidden="true"
+                        />
+                      }
+                    />
+                    {canViewContent && isSellingLive && (
+                      <>
+                        <div className="border-t border-gray-900/50 px-3 pb-3 pt-5 font-mono text-xs font-semibold uppercase tracking-wide">
+                          Account
+                        </div>
+                        <ul>
+                          {!canViewTeam && (
+                            <MobileNavLink path="/team" label="Invite team" />
+                          )}
```

Comment:
This should be `canViewTeam` instead of `!canViewTeam`.

```suggestion
                          {canViewTeam && (
                            <MobileNavLink path="/team" label="Invite team" />
                          )}
```